server/data/orchestrationApiTypes.ts

```diff
@@ -44,3 +44,5 @@ export type PrisonRegisterPrisonDto = components['schemas']['PrisonRegisterPriso
 export type VisitDto = components['schemas']['VisitDto']
 
 export type VisitorInfoDto = components['schemas']['VisitorInfoDto']
+
+export type BookerPrisonerVisitorRequestDto = components['schemas']['BookerPrisonerVisitorRequestDto']
```

Comment:
Unlike in the staff service, this file is in order - so suggest moving up a bit!